pymodbus/server/async_io.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, owner):
         self.receive_queue: asyncio.Queue = asyncio.Queue()
         self.handler_task = None  # coroutine to be run on asyncio loop
         self.framer: ModbusFramer
+        self.event_loop = asyncio.get_running_loop()
```

Comment:
Please call the variable self.loop to be consistent with the rest of the code.

Reply:
changed